src/imageio/imageio_exr.cc

```diff
@@ -140,6 +140,45 @@ dt_imageio_retval_t dt_imageio_open_exr(dt_image_t *img, const char *filename, d
       if(Imf::hasExpTime(header)) img->exif_exposure = Imf::expTime(header);
       if(Imf::hasAperture(header)) img->exif_aperture = Imf::aperture(header);
       if(Imf::hasIsoSpeed(header)) img->exif_iso = Imf::isoSpeed(header);
+#if(((OPENEXR_VERSION_MAJOR << 24) | (OPENEXR_VERSION_MINOR << 16) | (OPENEXR_VERSION_PATCH << 8)) >= 0x03020000)
+      if(Imf::hasCameraMake(header))
+      {
+        g_strlcpy(img->exif_maker, Imf::cameraMake(header).c_str(), sizeof(img->exif_maker));
+      }
+      if(Imf::hasCameraModel(header))
+      {
+        g_strlcpy(img->exif_model, Imf::cameraModel(header).c_str(), sizeof(img->exif_model));
+      }
+      // Make sure we copy the exif make and model to the correct place if needed
+      dt_image_refresh_makermodel(img);
+
+      if(Imf::hasLensModel(header))
+      {
+        g_strlcpy(img->exif_lens, Imf::lensModel(header).c_str(), sizeof(img->exif_lens));
+
+        if(Imf::hasLensMake(header) && Imf::lensMake(header) == "Canon")
+        {
+          /* Use pretty name for Canon RF & RF-S lenses (as exiftool/exiv2/lensfun) */
+          if(g_str_has_prefix(img->exif_lens, "RF"))
+          {
+            std::string pretty;
+            if(img->exif_lens[2] == '-')
+              pretty = "Canon RF-S " + std::string(&img->exif_lens[4]);
+            else
+              pretty = "Canon RF " + std::string(&img->exif_lens[2]);
+            g_strlcpy(img->exif_lens, pretty.c_str(), sizeof(img->exif_lens));
+          }
+        }
+
+        /* Capitalize Nikon Z-mount lenses properly for UI presentation */
+        if(g_str_has_prefix(img->exif_lens, "NIKKOR"))
+        {
+          for(size_t i = 1; i <= 5; ++i) img->exif_lens[i] = g_ascii_tolower(img->exif_lens[i]);
+        }
+      }
+
+      if(Imf::hasEffectiveFocalLength(header)) img->exif_focal_length = Imf::effectiveFocalLength(header);
```

Comment:
Not 100% sure about this one though, [`nominalFocalLength`](https://openexr.com/en/latest/StandardAttributes.html#lens-state) could be an option...

Reply:
Especially since it is said:

> The effective focal length is an abstraction used in lens design and, unless a CGI application is sophisticated enough to be using the thick lens model, should not be supplied to the application; for normal CGI applications, pinhole focal length should be used. 

So maybe better use the pinhole focal length if it exists ?

Reply:
Seems CGI specific though... I'm not convinced a real camera will fill this one in?

We could also leave focal length out for now until we see some real world usage...

Reply:
Your call, I have no strong opinion and you're lot more expert on this area that I am.

Reply:
After conferring w/ [upstream](https://github.com/AcademySoftwareFoundation/openexr/pull/1511#issuecomment-1713986390), let's go w/ "nominalFocalLength". Thanks.